src/splitwisesdk/core/ExpenseRatio.java

```diff
@@ -0,0 +1,25 @@
+package splitwisesdk.core;
+
+public class ExpenseRatio {
+
+    public People creditor = new People();
```

Comment:
Don't initialize empty instance of People here

Reply:
Instead add a public function
setCreditor(People people) {
    this.creditor = people;
}

same for debitor too

also add getter for both